src/ai/bot.test.js

```diff
@@ -282,6 +282,31 @@ describe('MCTSBot', () => {
     }
   });
 
+  test('objectivesWithWeightFunction', () => {
```

Comment:
Let's club this and the test above (objectives where the weight is specified as a number) under one `describe`?

Reply:
Makes sense - I will make that change